src/cache_test.ts

```diff
@@ -1460,4 +1460,29 @@ describe('delete items', () => {
 
         strictEqual(await connectPromise, true);
     });
+
+    it('should correctly handle errors in the initial list', async () => {
+        const fake = mock.mock(Watch);
+        const requestErr = Error('request failed');
+        const listFn: ListPromise<V1Namespace> = function (): Promise<V1NamespaceList> {
+            return new Promise<V1NamespaceList>((resolve, reject) => {
+                reject(requestErr);
+            });
+        };
+        const lw = new ListWatch('/some/path', fake, listFn);
+        let gotErr: Error | null = null;
+        const errCalled = new Promise<void>((resolve, reject) => {
+            lw.on('error', (err) => {
+                gotErr = err;
+                resolve();
+            });
+        });
+        try {
+            await lw.start();
+            await errCalled;
+            equal(gotErr, requestErr);
```

Comment:
Since we want a `===` check here:
```suggestion
            strictEqual(gotErr, requestErr);
```

Reply:
done (and removed the import at the top)